projects/max14906/src/examples/iio_example/iio_example.c

```diff
@@ -0,0 +1,117 @@
+/***************************************************************************//**
+ *   @file   max14906/src/examples/iio_example/iio_example.c
```

Comment:
do we really need such a long path here ?

Reply:
other files had the same. I guessed I should do the same.